scripts/platforms/workday_apply.py

```
@@ -0,0 +1,394 @@
+
+
+async def extract_field_context_generic(el):
+    name = (await el.get_attribute("name") or "").strip()
+    placeholder = (await el.get_attribute("placeholder") or "").strip()
+    aria = (await el.get_attribute("aria-label") or "").strip()
+    field_id = (await el.get_attribute("id") or "").strip()
+    label = ""
+    if field_id:
+        try:
+            label = normalize_space(await el.page.locator(f"label[for='{field_id}']").first.inner_text(timeout=800))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)